www/css/themes/main.css

```diff
@@ -1,7 +1,7 @@
 body {
-  --bg-color: black;
-  --txt-color: white;
-  --disabled-color: lightgray;
+  --bg-color: white;
+  --txt-color: black;
+  --disabled-color: darkgray;
 }
 
```

Comment:
The color of `a` links (both visited and unvisited) should be changed so it matches the dark theme

![image](https://user-images.githubusercontent.com/32238504/62150661-42e5a500-b307-11e9-9060-01d1fa592266.png)

Reply:
do you want the color to be this?

Reply:
That is the current color of visited links, and in my opinion is too close to the background color.